applications/GeoMechanicsApplication/tests/cpp_tests/test_utilities/geo_custom_condition.cpp

```diff
@@ -0,0 +1,83 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Aron Noordam
+//
+#include "custom_utilities/dof_utilities.h"
+#include "geo_custom_condition.h"
+
+namespace Kratos::Testing
+{
+
+void GeoCustomCondition::CalculateLeftHandSide(MatrixType& rLeftHandSideMatrix, const ProcessInfo& rCurrentProcessInfo)
+{
+    if (!mStiffnessMatrixSet) {
+        mStiffnessMatrix    = rLeftHandSideMatrix;
+        mStiffnessMatrixSet = true;
+    } else rLeftHandSideMatrix = mStiffnessMatrix;
```

Comment:
As in the comment in the test file, it would be more clear if we don't use the calculate as a setter the first time we use it, but add public (custom) setters for this custom condition. The same holds for the other 'calculates' in the custom condition and custom element.